src/production/models.py

```diff
@@ -54,15 +54,33 @@ def is_active(self):
         else:
             return False
 
+    @property
     def status(self):
         if self.assigned and not self.accepted and not self.completed:
-            return {'slug': 'assigned', 'friendly': 'Awaiting response'}
+            return "assigned"
         elif self.assigned and self.accepted and not self.completed:
-            return {'slug': 'accepted', 'friendly': 'Task accepted, underway'}
+            return "accepted"
         elif self.assigned and not self.accepted and self.completed:
-            return {'slug': 'declined', 'friendly': 'Task declined'}
+            return "declined"
         elif self.completed and not self.editor_reviewed:
-            return {'slug': 'completed', 'friendly': 'Completed on {0}<br/>Awaiting manager review'.format(
-                self.completed.strftime("%Y-%m-%d %H:%M"))}
+            return "completed"
         elif self.completed and self.editor_reviewed:
-            return {'slug': 'closed', 'friendly': 'Task closed'}
+            return "closed"
+
+    FRIENDLY_STATUSES = {
+            "assigned": "Awaiting response",
+            "accepted": "Task accepted",
+            "declined": "Task declined",
+            "completed": "Task completed",
+            "closed": "Task closed",
```

Comment:
might be desirable to have an `unassigned` status as well? (its the only hole I can find the `status` property

Reply:
I'm not so sure about an `unassigned` status, they are by default assigned, but wonder if we should add an `unknown` status encase someone messes with the data outside the worklflow?

Reply:
If it is always assigned why are we checking `if self.assigned...` over and over in the `status` property?

Reply:
It can be deleted in admin, its not an immutable field. Safest to check its not null.